core/src/main/java/org/mifos/mobilewallet/core/data/fineract/entity/payload/StandingInstructionPayload.kt

```diff
@@ -0,0 +1,85 @@
+package org.mifos.mobilewallet.core.data.fineract.entity.payload
+
+import android.os.Parcel
+import android.os.Parcelable
+
```

Comment:
Instead of overriding all these methods from `Parcelable`, you can simply Add `@Parcelize` annotation here (on top of `data class  StandingInstructionPayload`) then you can simply delete all overridden methods. This [article](https://medium.com/@BladeCoder/a-study-of-the-parcelize-feature-from-kotlin-android-extensions-59a5adcd5909) might help if you want to learn how it works.